src/components/SolidLoginForm.jsx

```diff
@@ -49,16 +74,19 @@ function SolidLoginForm(props) {
     return (
       <div className="login-form">
         <form onSubmit={handleLogin}>
-          <label id="idp-label" htmlFor="idp">Identity Provider:</label>
+          <label id="idp-label" htmlFor="idp">
+            Identity Provider/WebID:
+          </label>
           <input
             name="idp"
             type="text"
             id="idp"
-            placeholder="Identity Provider..."
+            placeholder="Identity Provider or WebID..."
```

Comment:
There is no need to add "..." at the end.

Reply:
I have removed the 3 dots in [commit 9115e7d](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/9115e7d9e9488a895eb00c81d9b13e1bc2645b63)